test/puppetlabs/puppetdb/http/index_test.clj

```diff
@@ -179,13 +190,13 @@
           (is (= results #{{:certname "host2"}})))))
 
     (testing "nodes"
-      (testing "query should return all nodes (including deactivated ones)"
+      (testing "query should return only active nodes"
         (doseq [query [["from" "nodes"]
                        "nodes {}"]]
           (is (= (set (mapv :certname (query-result method endpoint query {})))
                  #{"host1" "host2" "host3"}))))
 
-      (testing "broad regexp query should return all nodes"
+      (testing "broad regexp query should return all active nodes"
```

Comment:
it was a common feature request until we allowed it. Disallowing it would be a regression.